frontend/website/src/Head.re

```diff
@@ -57,19 +65,19 @@ let make = (~extra, ~filename, _children) => {
       <link
         rel="stylesheet"
         type_="text/css"
-        href="/static/css/normalize.css"
+        href={Links.Cdn.url("/static/css/normalize.css")}
       />
       <link rel="stylesheet" type_="text/css" href={filename ++ ".css"} />
```

Comment:
Which css are we putting on the CDN and which are we not?

Reply:
All of it if we can -- I see I missed the "filename.css" ones. It's a bit hard to do the ones that are siblings of the jobs pages and such so maybe we can leave those out. Do you think that's okay?

font.css is super important to be CDN-ed for example.

Reply:
I think it makes sense for the sibling css to live with the html and for the font.css to be special-cased, yeah